python/src/ir.cc

```diff
@@ -1778,6 +1782,13 @@ void init_triton_ir(py::module &&m) {
                    printingFlags);
              }
            })
+      .def("get_pipeline_str",
+           [](PassManager &self) {
+             std::string str;
+             llvm::raw_string_ostream os(str);
+             self.printAsTextualPipeline(os);
+             return str;
```

Comment:
is this used anywhere?

Reply:
No but I found myself adding this function quite frequently for debugging purposes. Should I remove it?

Reply:
it's fine to keep if you think it is useful